code-challenges.js

```diff
@@ -1,41 +1,151 @@
 // ASSESSMENT 3: Coding Practical Questions
 
 
-// --------------------1) Create a function that returns the first 10 numbers of the Fibonacci sequence in an array. Expected output: [1, 1, 2, 3, 5, 8, 13, 21, 34, 55]
-
-
-
-
-// --------------------2) Create a function that takes in an array and returns a new array of only odd numbers sorted from least to greatest.
+//1) Create a function that returns the first 10 numbers of the Fibonacci sequence in an array. Expected output: [1, 1, 2, 3, 5, 8, 13, 21, 34, 55]
+
+const fib = (num) => {
+
+    //start with array of 0 and 1
+    //start the for loop at index of 2 to use the first two numbers 0, 1 to start adding
+    //find the value of the index before and the index before before
+    //add those two values to get new value
+    //push new value into the newArray 
+    //start newArray with 1. then push new values from there
+      
+  let arr = [0, 1];
+  let newArray = [1];
+  
+  for(let i = 2; i <= num; i++) {
+    arr[i] = arr[i - 1] + arr[i - 2]; 
+    newArray.push(arr[i]);
+  }
+  
+  return newArray;
+}
+    
+// console.log(fib(10));
+     
```

Comment:
Looks good. This is a difficult problem. I wonder if you could refactor your setup to just have a single array rather than arr and newArray. Just a thought.